tests/primer/primer_tool.py

```diff
@@ -237,7 +237,8 @@ def _create_comment(
                 filepath = str(message["path"]).replace(
                     str(package_data.clone_directory), ""
                 )
-                comment += f"{package_data.url}/blob/{package_data.branch}{filepath}#L{message['line']}\n"
+                without_suffix = package_data.url.strip(".git")
```

Comment:
This won't do what you want it to do if the repo name end with g i or t or a combination of them. See `"https://github/a/myrepotigg.git".strip(".git") `

Reply:
I'd use `rsplit(".git") [0]` (sorry can't suggest I'm on mobile.

Reply:
yikes, thanks.